### PR TITLE
SRVCOM-1619: Add a note that Serverless does not support multi-tenancy

### PR DESCRIPTION
**Pull request by rh-max**
For versions 4.6+

https://issues.redhat.com/browse/SRVCOM-1619

Preview: https://deploy-preview-40881--osdocs.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html

### CODE REVIEW

**Comment by rh-max** [comment]
@rhuss @nainaz Could you PTAL?

**Comment by nainaz** [comment]
I think Multi Tenancy can be acheived using Multi cluster topology. 
so may be if we reword it as below.


`OpenShift Serverless currently cannot be used in a multi-tenant manner on the single cluster. `

Could use better wording . :) 
--

**Comment by nainaz** [comment]
Thank you. 
Looks good.

File: serverless/install/install-serverless-operator.adoc [hunk 1]
```diff
@@ -13,6 +13,11 @@ This guide walks cluster administrators through installing the {ServerlessOperat
 {ServerlessProductName} is supported for installation in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
 ====
 
+[NOTE]
+====
+{ServerlessProductName} currently cannot be used in a multi-tenant configuration on a single cluster.
+====
+
```
Comment (abrennan89) [inline serverless/install/install-serverless-operator.adoc:20]: We shouldn't have two notes right next to each other like this - maybe these two could be combined into a section with a title, something like "Supported configurations" or something and list them as bullets?

**Pull request merged**